src/Rest/Webhook.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Exan\Fenrir\Rest;
+
+use Discord\Http\Endpoint;
+use Discord\Http\Http;
+use Exan\Fenrir\Command\Helpers\InteractionCallbackBuilder;
+use Exan\Fenrir\Rest\Helpers\HttpHelper;
+use JsonMapper;
+use React\Promise\ExtendedPromiseInterface;
+
+/**
+ * @see https://discord.com/developers/docs/resources/webhook
+ */
+class Webhook
+{
+    use HttpHelper;
+
+    public function __construct(private Http $http, private JsonMapper $jsonMapper)
+    {
+    }
```

Comment:
```suggestion
    public function __construct(private Http $http, private JsonMapper $jsonMapper) {}
```
Another empty constructor